lib/diagnostic.rb

```diff
@@ -13,28 +13,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    (enum).inject { |sum, n| sum + n }
```

Comment:
Correct. 
BTW, `inject` and `reduce` are actually the exact same method in Ruby - one is an alias for the other.